pkg/apis/operator/v1alpha1/tektonpipeline_types.go

```diff
@@ -102,6 +101,12 @@ type PipelineProperties struct {
 	// This field will be removed, see https://github.com/tektoncd/operator/issues/1497
 	// originally this field was removed in https://github.com/tektoncd/operator/pull/1481
 	// there is no use with this field, just adding back to unblock the upgrade
+
+	// not in use, see: https://github.com/tektoncd/pipeline/pull/7789
+	// this field is removed from pipeline component
+	// keeping here to maintain the API compatibility
+	EnableTektonOciBundles *bool `json:"enable-tekton-oci-bundles,omitempty"`
```

Comment:
So in the next or later release we can directly remove this field right from the API, since we are setting this to nil in defaulting ?

Reply:
@PuneetPunamiya we never remove this from `v1alpha1` API